microsoftgraph/microsoft-graph-explorer-v4
# Enhancement/hint when unauthenticated

## Overview
Fixes #974 

After running more than 8 queries when unauthenticated, the coachmark below appears
![image](https://user-images.githubusercontent.com/45680252/142753732-e27da2a9-837b-4a2e-bf8f-511503ae9acd.png)
Hovering over the coachmark displays this bubble content:
![image](https://user-images.githubusercontent.com/45680252/142753757-3b275070-d4fd-4bb6-aff7-9cf0e90d5011.png)




### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions
Sign out of GE.
Run 9 sample queries
A coachmark appears which prompts the user to sign in to see real data
